api/editor/.prettier.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Switched to a JSON format because it was throwing errors as a javascript file for some reason.